src/Features/Core/Diagnostics/Analyzers/RudeEditUserDiagnosticAnalyzer.cs

```diff
@@ -24,7 +24,7 @@ public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics
         public override Task AnalyzeSyntaxAsync(Document document, Action<Diagnostic> addDiagnostic, CancellationToken cancellationToken)
         {
             // No syntax diagnostics produced by the EnC engine.  
-            return Task.FromResult(false);
+            return SpecializedTasks.EmptyTask;
```

Comment:
It is very likely a safe change. I'm saying about a remote possibility. I'd review immediate callers to make sure they do not depend on the exact type, and/or find who wrote this line to ask about their intentions.